pages/Types.html

```diff
@@ -657,6 +658,8 @@ <h2 id="XMLHttpRequest"> XMLHttpRequest </h2>
 <h2 id="jqXHR"> jqXHR </h2>
 <p>As of jQuery 1.5, the <a href="/jQuery.ajax/">$.ajax()</a> method returns the jqXHR object, which is a superset of the XMLHTTPRequest object. For more information, see the <a href="/jQuery.ajax/#jqXHR">jqXHR section of the $.ajax entry</a>
 </p>
+<h2 id="Thenable">Thenable</h2>
+<p>Any object with a <code>then</code> method that returns a <a href="#Promise">Promise</a>-compatible object.</p>
```

Comment:
We should stick to the definition at https://promisesaplus.com/#point-7 i.e. the only assumption about `then` is that it's a function.